src/adapter.ts

```diff
@@ -0,0 +1,50 @@
+import { v4 as uuidv4 } from 'uuid'
+import User from './models/user'
+import {
+  createFolderIfNotExist,
+  createContent,
+  getContent,
+  deleteContent,
+} from './workingWithData'
+
+type UserInfoType = { name: string; age: number }
+
+class WorkingWithData {
+  pathToFile: string
+  fileName: string
+  //content: UserInfoType
+
+  // private readonly BodyaLox:string
+
+  constructor(pathToFile: string, fileName: string) {
+    this.pathToFile = pathToFile
+    this.fileName = fileName
+    // this.content = content
+  }
+
+  async init(data: UserInfoType) {
```

Comment:
you need also specify what your function will return. in this case you don't return anything, so you write this
`async init(data: UserInfoType): Promise<void> {`

Reply:
Please add return value for all functions below
https://stackoverflow.com/questions/59664670/typescript-async-function-return-type-void-vs-promisevoid****

Reply:
Done